draft-davidson-pp-architecture.md

```diff
@@ -54,11 +54,11 @@ informative:
       -
         ins: Gary Belvin
         org: Google
-  TRUST:
-    title: Trust Token API
-    target: https://github.com/WICG/trust-token-api
+  TrustTokenAPI:
```

Comment:
I'd leave this pointing at the WICG API (the explainer there and the spec, whenever I get around to actually publishing it) is probably the canonical for Trust Token.